noxfile.py

```diff
@@ -0,0 +1,25 @@
+import nox
```

Comment:
```suggestion
"""A nox configuration file so that we can build the documentation easily with nox.
- see the README.md for information about nox.
- ref: https://nox.thea.codes/en/stable/
"""
import nox
```